supertokens/supertokens-postgresql-plugin
# feat: support access token to jwt

## Summary of change
feat: support access token to jwt

## Related issues
- https://github.com/supertokens/supertokens-plugin-interface/pull/59
- https://github.com/supertokens/supertokens-core/pull/566

## Test Plan
Manual testing:
- run test creating a session with old core
- execute migration instruction
- check that new core can verify old session + sign new session

## Documentation changes
Done separately

## Checklist for important updates
   - In `build.gradle`
   - To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
   - If no such branch exists, then create one from the latest released branch.
